scripts/linux_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash +x
+
+BINARY_URL="https://github.com/massalabs/thyra/releases/latest/download/thyra-server_linux"
+SCRIPT="Linux"
+
+green () { echo -e "\033[01;32m$1:\033[0m $2"; }
+
+warn () { echo -e "\033[01;33m[$SCRIPT]WARNING:\033[0m $1"; }
+
+fatal () { echo -e "\033[01;31m[$SCRIPT]ERROR:\033[0m $1" >&2; exit 1; }
+
+architecture_version () {
+    case $(uname -m) in
+        x86_64) echo "amd64" ;;
+        # arm64)  echo "arm64" ;; ARM64 is unsupported on Linux for now
+        *)      fatal "$(uname -m) unsupported." ;;
+    esac
+}
+
+check_supported_distro () {
+    grep '^ID.*=.*ubuntu' /etc/os-release &> /dev/null || fatal "Aborting: You are using an unsupported distribution."
+}
+
+install_thyra () {
+    arch="$(architecture_version)" || exit 1
```

Comment:
could it exit without error message? should we do here `fatal` like at other places?

Reply:
There is no need for an error message here since it's already printed [here](https://github.com/massalabs/thyra/blob/0f345d56268b5f7eace23fe8c08a3b2de9b4f801/scripts/linux_install.sh#L16).

Reply:
Okey then I think that `|| exit 1` is not necessary

Reply:
If we remove the `|| exit 1`, the script won't stop at that line and will continue to run the next steps of the function. Those `exit 1` are like `return`.

Reply:
I said that because there is an `exit 1` in the `fatal` function